### PR TITLE
Adds support for WebVR mappings

### PR DESCRIPTION
**Pull request by NellWaliczek**
This change moves existing mappings into a WebXR subfolder and renames
them to conform with WebXR naming conventions. It then adds a WebVR
subfolder in which new WebVR mappings are added based on the known
gamepad ids in released browsers. The schema is extended to add an
property indicating a mapping is for WebVR. When this property is true,
additional WebVR-specific properties for targetRay origin and axis
inversion are added to the valid schema.
Tests are added and test tools are modified to look for  mapping files
in both WebVR and WebXR subfolders.

### CODE REVIEW

**Comment by NellWaliczek** [comment]
Fixes #5

File: tests/testHelpers.js [hunk 1]
```diff
@@ -29,9 +31,9 @@ const TestHelpers = {
    * be searched instead of the real mappings folder
    * @returns {Object} The mapping described in the mapping.json file
    */
-  getMappingById : function (gamepadId, useMocks) {
-    let folder = useMocks ? mockMappingsFolder : mappingsFolder;
-    let mappingPath = join(folder, gamepadId, "mapping.json")
+  getMappingById : function (gamepadId, mappingType = Constants.MappingType.WEBXR) {
```
Comment (NellWaliczek) [inline tests/testHelpers.js]: Fix function metadata

File: mappings/WebVR/Oculus Go Controller/mapping.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
+{
+    "version" : "0.1.0",
```
Comment (toji) [inline mappings/WebVR/Oculus Go Controller/mapping.json]: I assume you want these to be just "0.1" to match with #35?

**Comment by toji** [comment]
Didn't verify all the mappings 😉 but the rest looks good with one question about the version.

File: mappings/WebVR/Oculus Go Controller/mapping.json (hunk 2 above)
Comment (NellWaliczek) [reply mappings/WebVR/Oculus Go Controller/mapping.json]: Yeah, I wasn't sure which would get approved first ;)

**Pull request merged**